### PR TITLE
Rain 1241

### PR DESCRIPTION
**Pull request by rohit-eGov**

### CODE REVIEW

File: pdf-service/LOCALSETUP.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- [ ] Elasticsearch
+- [x] Kafka
+  - [x] Consumer
+  - [x] Producer
+
+## Running Locally
+
+### Local setup
+1. To setup the PDF-service, clone the [Core Service repository](https://github.com/egovernments/core-services)
+2. Open the terminal and run the following command
```
Comment (tarunlalwani) [inline pdf-service/LOCALSETUP.md:20]: which command?

File: pdf-service/LOCALSETUP.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    
+    `DATA_CONFIG_URLS: process.env.DATA_CONFIG_URLS || "file:///home/xyz/Documents/configs/pdf/data/abc-data.json"`
+
+     `FORMAT_CONFIG_URLS: process.env.FORMAT_CONFIG_URLS || "file:///home/xyz/Documents/configs/pdf/format/abc-format.json"`
+
+4. Open the terminal and run the following command
+
+    `cd [filepath to pdf service]`
+                                                      
+    `sudo npm install --unsafe-perm=true`             (run this command only once when you clone the repo)
```
Comment (tarunlalwani) [inline pdf-service/LOCALSETUP.md:33]: Why `sudo` is needed? It should not be required/used at all. Also why are we using `--unsafe-perm=true`?

**Review by tarunlalwani — CHANGES_REQUESTED** [review]

File: pdf-service/LOCALSETUP.md (hunk 1 above)
Comment (rohit-eGov) [reply pdf-service/LOCALSETUP.md]: Updated the localsetup file, this line was added by mistake

File: pdf-service/LOCALSETUP.md (hunk 2 above)
Comment (rohit-eGov) [reply pdf-service/LOCALSETUP.md]: Updated the command

**Review by tarunlalwani — APPROVED** [review]

**Pull request merged**